app/views/admin/posts/_form.html.haml

```diff
@@ -1,40 +1,46 @@
 
-  - if defined?(EffectiveArticleEditor)
-    = f.article_editor :excerpt, hint: 'Will be used for the post excerpt on index pages.'
-  - else
-    = f.rich_text_area :excerpt, hint: 'Will be used for the post excerpt on index pages.'
+      - if defined?(EffectiveArticleEditor)
+        = f.article_editor :excerpt, hint: 'Will be used for the post excerpt on index pages.'
+      - else
+        = f.rich_text_area :excerpt, hint: 'Will be used for the post excerpt on index pages.'
 
-  = f.text_field :description, hint: 'The content of the post meta tags.', maxlength: 150
+      = f.text_field :description, hint: 'The content of the post meta tags.', maxlength: 150
 
-  - if Array(EffectivePosts.categories).length > 1
-    = f.select :category, EffectivePosts.categories
-  - else
-    = f.hidden_field :category, value: (EffectivePosts.categories.first || 'posts')
+      - if Array(EffectivePosts.categories).length > 1
+        = f.select :category, EffectivePosts.categories
+      - else
+        = f.hidden_field :category, value: (EffectivePosts.categories.first || 'posts')
 
-  - if EffectivePosts.use_active_storage
-    = f.file_field :image, hint: 'An image for your post'
+      - if EffectivePosts.use_active_storage
+        = f.file_field :image, hint: 'An image for your post'
 
-  - if defined?(EffectiveArticleEditor)
-    = f.article_editor :body, hint: 'The main body of your post'
-  - else
-    = f.rich_text_area :body, hint: 'The main body of your post'
+      - if defined?(EffectiveArticleEditor)
+        = f.article_editor :body, hint: 'The main body of your post'
+      - else
+        = f.rich_text_area :body, hint: 'The main body of your post'
 
-  = render partial: '/effective/posts/additional_fields', locals: { post: post, form: f, f: f }
+      = render partial: '/effective/posts/additional_fields', locals: { post: post, form: f, f: f }
 
-  = f.datetime_field :published_at, label: 'Publish date', hint: 'When should this be displayed on the website.'
+      = f.datetime_field :published_at, label: 'Publish date', hint: 'When should this be displayed on the website.'
 
-  = f.check_box :draft, hint: 'Save this post as a draft. It will not be accessible on the website.'
+      = f.check_box :draft, hint: 'Save this post as a draft. It will not be accessible on the website.'
 
-  - if defined?(EffectiveRoles) && f.object.respond_to?(:roles) && EffectivePosts.use_effective_roles
-    = render partial: '/admin/posts/roles', locals: { post: post, form: f, f: f }
+      - if defined?(EffectiveRoles) && f.object.respond_to?(:roles) && EffectivePosts.use_effective_roles
+        = render partial: '/admin/posts/roles', locals: { post: post, form: f, f: f }
+
+    - if respond_to?(:acts_as_tagged?)
```

Comment:
This is checking the view helper, which we need to remove.  This check should be

```
- if f.object.class.try(:acts_as_tagged?)
  = render('effective/tags/fields, f: f)
```